src/mysqlsh_helpers.py

```diff
@@ -19,6 +22,31 @@ class MySQL:
     def __init__(self):
         pass
 
+    @property
+    def mysqlsh_bin(self):
+        """Determine binary path for MySQL Shell.
+
+        :returns: Path to binary mysqlsh
+        :rtype: str
+        """
+        # Allow for various versions of the mysql-shell snap
+        # When we get the alias use /snap/bin/mysqlsh
+        if os.path.exists("/snap/bin/mysqlsh"):
+            return "/snap/bin/mysqlsh"
+        if os.path.exists("/snap/bin/mysql-shell.mysqlsh"):
+            return "/snap/bin/mysql-shell.mysqlsh"
+        # Default to the full path version
+        return "/snap/bin/mysql-shell"
+
+    @property
+    def mysqlsh_common_dir(self):
+        """Determine snap common dir for mysqlsh
+
+        :returns: Path to common dir
+        :rtype: str
+        """
+        return "/root/snap/mysql-shell/common"
```

Comment:
will we always run `mysqlsh` as `root`? it is downloaded as `root`, but i wonder if we should do things with user `mysql` in a future PR